### ISSUE TITLE
Integrate HMMR

### ISSUE THREAD

**Issue by tfederico** [akanazawa/motion_reconstruction#11]
Hi,

first of all, thanks for releasing the code.

Do you have any suggestion about how to integrate HMMR instead of HMR with this code?
Something like a todo list or steps to go through

**Comment by sxhzju** [comment]
bump bump:smile:

**Comment by acoMCMXCVI** [comment]
Hi @tfederico! Did you find a way to integrate HMMR? :)

**Comment by tfederico** [comment]
no, I skipped the motion reconstruction phase as HMMR is giving me results that are good enough

**Comment by acoMCMXCVI** [comment]
Ok, thank you!
